### PR TITLE
Adds storybook to ace

### PR DESCRIPTION
**Pull request by djblue**
@adimka 
@andrewkfiedler 
@samuelechu 
@gordocanchola

### CODE REVIEW

File: bin.js [hunk 1]
```diff
@@ -121,6 +121,14 @@ program
...
+program
+  .command('storybook')
+  .description('start the storybook server')
+  .option('--port <port>', 'dev server port (default: 8080)')
+  .option('--host <host>', 'dev server host (default: localhost)')
+  .option('--static', 'Build static version of storybook.')
+  .action(wrap('./lib/storybook'))
+
 program
   .command('disable-idp')
```
Comment (Bdthomson) [inline bin.js]: Remove in a new pr?

File: lib/webpack.config.js [hunk 2]
```diff
@@ -217,6 +217,103 @@ const base = ({ alias = {}, env, tsTranspileOnly }) => ({
...
+            loader: nodeResolve('exports-loader'),
+            options: { Cesium: true },
+          },
+          nodeResolve('script-loader'),
+        ],
+      },
+      {
+        test: /\.jsx?$/,
+        exclude: function(modulePath) {
+          if (modulePath.indexOf('catalog-ui-search') > -1) {
```
Comment (Bdthomson) [inline lib/webpack.config.js]: `ace` is referencing a downstream package, that doesn't seem right.

File: bin.js (hunk 1 above)
Comment (djblue) [reply bin.js]: I think we still need this :man_shrugging:

File: lib/webpack.config.js [hunk 3]
```diff
@@ -217,6 +217,106 @@ const base = ({ alias = {}, env, tsTranspileOnly }) => ({
...
+          },
+          {
+            loader: nodeResolve('react-docgen-typescript-loader'),
+            options: {
+              skipPropsWithoutDoc: true,
+              // The `transpileOnly` option is set to true because the
+              // storybook environment is meant for rapid iteration and
+              // not validation. We should not be using storybook to
+              // enforce typing.
+              transpileOnly: true,
```
Comment (Bdthomson) [inline lib/webpack.config.js]: This is the wrong block. This should be in the options for `ts-loader`.

Comment (djblue) [reply lib/webpack.config.js]: :man_facepalming:

**Comment by Bdthomson** [comment]
Have been using this for about a day without any issues. Searches recursively for stories and starts up quick.

**Comment by Bdthomson** [comment]
Some would say it's the best PR of all time

**Pull request merged**